src/App.tsx

```diff
@@ -8,7 +8,76 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { SortType } from './types/FilterTypes';
+
+type ReorderOptions = {
+  filterType: SortType,
+  query: string,
+};
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todosLoaded, setTodosLoaded] = useState(true);
+
+  const [selectedTodo, setSelectedTodo] = useState<number | null>(null);
+
+  const [query, setQuery] = useState('');
+  const [filterType, setFilterType] = useState(SortType.All);
+
+  async function LoadTodos() {
+    const reponse = await getTodos();
+
+    setTodosLoaded(false);
+
+    setTodos(reponse);
+  }
+
+  useEffect(() => {
+    LoadTodos();
+  }, []);
+
+  const onChangeFilterType = (type: SortType) => {
```

Comment:
Actually, seems like now we don't need this wrapper at all. you can pass `setFilterType` to `onChangeFilterType` prop

`onChangeFilterType={setFilterType}`